appgate/files.py

```diff
@@ -119,7 +119,7 @@ def load_file(self) -> str:
                 response.release_conn()
         except Exception as e:
             raise AppgateFileException(
-                "Unable to fetch the file contents for %s: %s", e
+                "Unable to fetch the file contents for %s/%s: %s", bucket, object_key, e
```

Comment:
Does `AppgateFileException` do lazy string formatting?

Reply:
I dont think so, it's just a normal Exception